packages/common/pipes/parse-float.pipe.ts

```diff
@@ -48,6 +65,21 @@ export class ParseFloatPipe implements PipeTransform<string> {
         'Validation failed (numeric string is expected)',
       );
     }
-    return parseFloat(value);
+
+    const float = parseFloat(value);
+
+    if (this.min != null && float < this.min) {
```

Comment:
I guess this would be better for readability:

```suggestion
    if (!isNil(this.min) && float < this.min) {
```

`import { isNil } from '../utils/shared.utils';`